### ISSUE TITLE
Windows detection not working 100%

### ISSUE THREAD

**Issue by kriegfrj** [GabrielBB/xvfb-action#11]
I just found this action and it is exactly what I needed - thanks!

However, I found that the windows detection wasn't working properly - I had to split out Windows from the build matrix. I suspect this is because it's using cygwin or some Unix-like shell. See bndtools/bnd#4163.

This error is reported against 1.2. I have since had to downgrade it to 1.0 due to #10.

**Comment by kriegfrj** [comment]
I have just verified that there are still problems with the auto-detection on windows with 1.0 - see the following output (from https://github.com/kriegfrj/bnd/actions/runs/146783355):

```
Unable to locate executable file: d:\a\bnd\bnd\.github\scripts\build.sh. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also verify the file has a valid extension for an executable file.
```

The same command works fine when run without `xvfg-action`.

**Comment by GabrielBB** [comment]
Hi @kriegfrj The problems with 1.2 were fixed in [v1.4](https://github.com/marketplace/actions/gabrielbb-xvfb-action). Maybe you can test using `working_directory` with this new version?

**Comment by GabrielBB** [comment]
`v1` tag updated
